style.css

```diff
@@ -19,24 +19,18 @@ body {
   max-height: max-content;
 }
 
+#nav-desktop {
+  display: none;
+}
+
 #menu-mobile {
```

Comment:
- [ ] One of the requirements says that when building the desktop version your mobile version should remain the same But in your design I have fount out an extra space at the right of your design kindly fix it. check here 👇 

![image](https://user-images.githubusercontent.com/65068771/150425270-cd1ad7c6-858b-41ba-be71-be58d5556f47.png)

- [ ] Note that you need to use the same font family of the design(`font-family: Roboto`). check [her](https://fonts.google.com/specimen/Roboto) and see how you can add a external font-family to your project